Base/QTGUI/qSlicerDarkStyle.cxx

```diff
@@ -41,31 +41,31 @@ QPalette qSlicerDarkStyle::standardPalette()const
 
   // See https://doc.qt.io/qt-5/qpalette.html#ColorRole-enum
   // Central color roles
-  palette.setColor(QPalette::Window, "#353535");
+  palette.setColor(QPalette::Window, "#323232");
   palette.setColor(QPalette::WindowText, Qt::white);
-  palette.setColor(QPalette::Disabled, QPalette::WindowText, "#7f7f7f");
-  palette.setColor(QPalette::Base, "#2d2d2d");
-  palette.setColor(QPalette::Disabled, QPalette::Base, "#323234");
-  palette.setColor(QPalette::AlternateBase, "#424242");
+  palette.setColor(QPalette::Disabled, QPalette::WindowText, "#6d6d6d");
+  palette.setColor(QPalette::Base, "#1e1e1e");
+  palette.setColor(QPalette::Disabled, QPalette::Base, "#2d2d2d");
```

Comment:
It appear in the macOS dark mode control, the disabled base color is the same as the base color if we want to match this state as well. This can be seen in the current screenshots of the Models module.

Reply:
Do I understand correctly saying that:
* on Linux and Windows, we will be using our custom qSlicerDaskStyle
* on macOS, we will *not* have an explicit dark mode available and we will let the operating system taking care of it ?

And finally, when you asked "It appear in the macOS dark mode control, the disabled base color is the same as the base color if we want to match this state as well", it means that we will update our own `qSlicerDarkStyle` palette to that Linux and Windows dark theme match macOS system one ?

Reply:
This PR does not remove the Style selection for macOS, but that is something we should do as the latest Qt version we are using now supports letting macOS control the dark mode based on if the OS is using light or dark.  This PR just adjusts the qSlicerDarkStyle so that Linux and Windows can have a similar experience to what users on macOS will have automatically.